src/pkg.cc

```diff
@@ -57,6 +59,126 @@ UInt T_SEMI   = 0;
 UInt T_BIPART = 0;
 UInt T_BLOCKS = 0;
 
+// Forward decl
```

Comment:
Maybe this?
```suggestion
// Forward decl of a function imported from the GAP kernel
```
I wonder how the code even loads with this in older GAP versions, though?!? Anyway, with the `#if` check for kernel versions, this would be a non-issue.

Reply:
Right you are, deleted!